ajax/getDropdownValue.php

```diff
@@ -243,6 +243,8 @@
              ORDER BY $add_order `$table`.`completename`
              $LIMIT";
 
+   $datastoadd = array();
```

Comment:
yeah, in fact, it's not a part of this pr, juste a residual bug .
I can remove this part and fix it in upstream